.circleci/config.yml

```diff
@@ -67,6 +95,64 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  e2e-simple-cni:
+    <<: *integrationDefaults
+    environment:
+            - KUBECONFIG: /go/out/minikube.conf
+            - TEST_ENV: minikube-none
+            - GOPATH: /go
+    steps:
+      - <<: *initWorkingDir
+      - checkout
+      - attach_workspace:
+          at:  /go
+      - <<: *markJobStartsOnGCS
+      - run: make sync
+      - run: make build
+      - run:
+          command: |
+            if [ ! -f /go/out/linux_amd64/release/istio-cni ]; then
+              # Should only happen when re-running a job, and the workspace is gone
+              time make build
+            fi
+            make docker.all
+      - run: docker images
+      - run:
+          name: Get Istio
+          shell: /bin/bash
+          working_directory: /go/src/istio.io/istio
+          command: |
+            git clone https://github.com/tiswanso/istio.git .
+            git checkout istio-cni_helm
+      - run: /go/src/istio.io/istio/bin/testEnvRootMinikube.sh start
+      - run: /go/src/istio.io/istio/bin/testEnvRootMinikube.sh wait
+      - run: docker pull gcr.io/istio-release/galley:master-latest-daily
+      - run: docker images
+      - run:
+          name: Apply Istio CRDs
+          shell: /bin/bash
+          working_directory: /go/src/istio.io/istio
+          command: |
+            kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
```

Comment:
The tests do this step so it should be needed.

Reply:
The tests don't do it as a separate step so there's a timing issue that I see all the time without this step.  I'm pretty sure the main reason is because the istio/istio circleci tests all build the docker images locally and don't need to pull but in istio-cni we just want to grab a known good istio/istio hub+tag and not build images locally.